data/remnant/remnant 2 cognizance.txt

```diff
@@ -53,14 +53,13 @@ mission "Remnant: Cognizance 1"
 		personality entering forbearing timid uninterested surveillance staying
 		fleet
 			names "korath"
-			cargo 1
-			commodities "void sprite parts"
```

Comment:
Might be worth noting that there *is* a "void sprite parts" commodity list (see line 869 in `commodities.txt`), and my intent was that this pulls one option from that list to fill a random amount of cargo space.

Having a random 3 cargo doesn't have the same impact.

And yes, while having some void sprite parts is just a junk cargo for the moment, these are intended to be used for future side missions.

Reply:
It's not just that it adds "junk cargo." It is totally ignored.
If it were given a price range like that normal trade commodities, it would be possible for it to show up here, but that would also mean it appears in the trade tab on every planet that has one, which is nonsensical.

![image](https://user-images.githubusercontent.com/20605679/191547321-3d6766cf-daf6-453d-8d40-6647de045023.png)

Reply:
I have given them all `cargo 0`, so now they won't have any commodities (and can't have outfits because there are no outfitters near where they spawn).